sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/SampleSnippets.cs

```diff
@@ -30,6 +32,20 @@ public void CreateFormRecognizerClient()
             #endregion
         }
 
+        [Test]
+        public void CreateFormTrainingClient()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+
+            #region Snippet:CreateFormTrainingClient
+            //@@ string endpoint = "<endpoint>";
```

Comment:
Do you guys include the "using" statements in the snippets?

Reply:
not really. We didn't for TA and looking around other libraries don't see that as a pattern